$MirOS: ports/x11/gnome/applets2/patches/patch-configure_in,v 1.1.7.1 2005/03/18 16:00:21 tg Exp $
--- configure.in.orig	Mon Mar 12 13:56:46 2007
+++ configure.in	Tue Jul  3 22:47:32 2007
@@ -40,14 +40,14 @@ dnl ************************************
 AM_MAINTAINER_MODE
 
 IT_PROG_INTLTOOL([0.35.0])
-PKG_PROG_PKG_CONFIG([0.19])
+PKG_PROG_PKG_CONFIG
 
 AC_PROG_CC
 AC_ISC_POSIX
 AC_STDC_HEADERS
 AM_PROG_LIBTOOL
 AC_PATH_XTRA
-AM_PATH_PYTHON(2.4)
+AM_PATH_PYTHON(2.3)
 AM_CHECK_PYTHON_HEADERS(HAVE_PYHDRS="yes", HAVE_PYHDRS="no")
 
 X_LIBS="$X_LIBS $X_PRE_LIBS -lX11 $X_EXTRA_LIBS"
@@ -383,7 +383,7 @@ if test x$disable_battstat = xno; then
 	   AC_CHECK_LIB(apm,apm_read,[HAVE_LIBAPM=yes],[HAVE_LIBAPM=no])
 	   ;;
        # list of supported OS cores that do not use libapm
-       i386-*-freebsd*|*-*-netbsd*|*-*-openbsd*|*-*kfreebsd*-gnu)
+       i386-*-freebsd*|*-*-netbsd*|*-*-openbsd*|*-*-mirbsd*|*-*kfreebsd*-gnu)
           if [ -n "${OS_SYS}" ]; then
 	       ACPIINC="-I${OS_SYS}"
 	   else
